packages/design-system/src/styles/components/_TooltipIcon.scss

```diff
@@ -40,13 +42,13 @@
 .ds-c-tooltip__trigger:hover,
 .ds-c-tooltip__trigger:focus {
   .ds-c-tooltip-icon__container {
-    border-color: $tooltip-icon__color;
-    border-color: rgba($tooltip-icon__color, 0.25);
+    border-color: var(--tooltip-icon__color);
+    border-color: rgba(var(--tooltip-icon__color), 0.25);
```

Comment:
this color function shouldn't work with css vars - there's goofiness in how the compiled var value fits in with this function. 

sass allows for these color functions to work with sass vars, css props don't.

Reply:
[SO explains it better than me](https://stackoverflow.com/a/41265350)

Reply:
Interesting, and too bad! I've updated these variables to use new token variables which are computed during compilation.